src/modules/DisplayModule.cpp

```diff
@@ -0,0 +1,49 @@
+#include "DisplayModule.h"
+#include <cstdio>
+
+DisplayModule::DisplayModule(ModularSynth& synth)
+	:SynthModule(synth, moduleId, 1, 1, 0)
+{
+}
+
+
+void DisplayModule::cycle()
+{
+	sprintf(mInputValue, "%.2f", getInput(0));
```

Comment:
Just noticed this and I would fix this myself but maybe you will find the following useful (good practice for me as well how to work with PR's). ;) 

Generally, you will want to do this in the render method. This updates 44100 times a second while the render method updates only when rendering the module, which is up to something like 60 times per module.

I suggest you move this to render(), however you then have to either render the module content yourself (not that hard, really) or check e.g. EffectModule how it updates its label and then overrides getName() to pass the label to the parent render method.